src/directives/class-map.ts

```diff
@@ -48,7 +48,7 @@ export const classMap = directive((classInfo: ClassInfo) => (part: Part) => {
 
   // handle static classes
   if (!classMapCache.has(part)) {
-    element.className = committer.strings.join(' ');
+    element.classList.value = committer.strings.join(' ');
```

Comment:
Sorry, should have been more clear. We already have a `classList` variable just below that we can reusue here if we move it up.

Reply:
😅 My mistake. I should have looked a bit more at the context.

Reply:
Fixed.